server/application/application.proto

```diff
@@ -110,7 +110,19 @@ message ApplicationResourceDeleteRequest {
 	optional bool force = 7 [(gogoproto.nullable) = true];
 }
 
+message ResourceActionRunRequest {
+	required string name = 1;
+	required string namespace = 2 [(gogoproto.nullable) = false];
+	required string resourceName = 3;
+	required string version = 4 [(gogoproto.nullable) = false];
+	required string group = 5 [(gogoproto.nullable) = false];
+	required string kind = 6 [(gogoproto.nullable) = false];
+	required string action = 7 [(gogoproto.nullable) = false];
+}
```

Comment:
Just curious why is resourceName not `[(gogoproto.nullable) = false]` like the other resource specific requests. e.g.:
```
message ApplicationResourcePatchRequest {
	required string name = 1;
	required string namespace = 2 [(gogoproto.nullable) = false];
	required string resourceName = 3 [(gogoproto.nullable) = false];
	required string version = 4 [(gogoproto.nullable) = false];
	required string group = 5 [(gogoproto.nullable) = false];
	required string kind = 6 [(gogoproto.nullable) = false];
	required string patch = 7 [(gogoproto.nullable) = false];
	required string patchType = 8 [(gogoproto.nullable) = false];
}
```